backend/src/socket/README.md

```diff
@@ -17,18 +17,18 @@
 
 ### Listen
 
-| function         | event name       | parameters      | description                       |
-| ---------------- | ---------------- | --------------- | --------------------------------- |
-| handleSetCanvas  | setCanvas        | client, payload |                                   |
-| handleFindGame   | findGame         | client          | User try to join or create a game |
-| handleStartGame  | startGame        | client, payload | the gameInterval starts           |
-| handleMovePaddle | movePaddle       | client, payload |
-| handeInvitePlayer| invitePlayer			| client, payload | User invites player
-|									 |									|	{userName}			| 											            |
-| handleRespondTo	 | respondToInvite	| client, payload | User invites player
-|	Invite					 |									|	{accept, user-	| User accepts or declines invitation
-|									 |									|	Name, gameID}		|
-|                  | pauseGame        |                 |                                   |
+| function          | event name      | parameters      | description                         |
+| ----------------- | --------------- | --------------- | ----------------------------------- |
+| handleSetCanvas   | setCanvas       | client, payload |                                     |
```

Comment:
This is to inform front, right? The function is not interesting to them, I think. While it is interesting to know what should be in the payload exactly. I would suggest deleting the first column and clarifying the 'parameters' column to make things more clear.

Reply:
Em....this is for receiving the canvase data from the front.
Sure it's better to clarify what parameters should be used.
The first column is to let the front end know what is the event name to send.

Reply:
Wait, I think you misunderstood. Maybe I didn't explain very clearly. I was talking about the setup of the whole list, just not about the `setCanvas()` function.

If this is info for the front, they need to know:
- The name of the event they have to send (`setCanvas`, `findGame`, `startGame`, etc)
- The exact names of the variables (and their respective type) that they need to pass in the payload object for the function to work correctly (`userName`, `gameID`, `accept`, etc.)

They do not need to know:
- The name of the function that handles the payload in the back (`handleSetCanvas`, `handleFindGame`, `handleStartGame`, etc) - or, the first column
- That the back receives a client object (since they don't need to send it)
- That the back receives a payload object that holds the sent variables